### PR TITLE
Development

### PR DESCRIPTION
**Pull request by BalintMendli**
Please review.

### CODE REVIEW

**Review by zlayabekrija — CHANGES_REQUESTED** [review]
Great first try, but there are many improvements needed.
Please use the floats instead of hardcoding the absolute position.
Most of the div that you have used absolute position with margins, should be done with the floats, and margin of auto.
Please adjust your project, and resubmit review request.
Happy coding.

**Comment by BalintMendli** [comment]
Made the requested changes. Please review.

Thank you.

File: assets/css/style.css [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+}
+
+.hero {
+  margin-left: 80px;
+  height: 530px;
+  width: 100%;
+  background-color: white;
+  display: inline-block;
+}
+.post1 {
```
Comment (zlayabekrija) [inline assets/css/style.css]: all three of this (post1, post2, post3) should be one class, something like `post {
float: left;
width: 33%;
height: 30vh;
}`

File: index.html [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+      <div class="post2">
+        <div class="pTitle"></div>
+        <div class="pText"></div>
+      </div>
+      <div class="post3">
+        <div class="pTitle"></div>
+        <div class="pText"></div>
+      </div>
+    </section>
+    <!-- <section class="latest">
```
Comment (zlayabekrija) [inline index.html]: Please don't keep commented out html

File: index.html [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+    </section>
+    <section class="person">
+      <div class="person-img">
+        <div class="person-img-inner"></div>
+      </div>
+      <div class="person-text">
+        <div class="person-text-inner"></div>
+      </div>
+    </section>
+    <!-- <section class="hero">
```
Comment (zlayabekrija) [inline index.html]: same here

File: index.html [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+      </div>
+    </section>
+    <section class="preFooter">
+      <div class="book"></div>
+      <div class="colorBar">
+        <div class="colBarText"></div>
+      </div>
+    </section>
+    <section class="hero">
+      <!-- <div class="text"></div>
```
Comment (zlayabekrija) [inline index.html]: delete please

**Review by zlayabekrija — CHANGES_REQUESTED** [review]
Good work, but the point of this project is different. Please explore more floats and positioning. With the floats, the absolute position becomes unnecessary as well as this large margins that you are using. An absolute position should be used only to adjust minor things on the screen. (e.g. keep div centered). Also, keep in mind that classes are made in order to be reused so classes post1,2,3 should do one and the same, it should be only one of them.

If you need any help, feel free to contact me on slack.

**Comment by BalintMendli** [comment]
Sorry, forgot to actually push my changes.
Please check again.
Thanks.

**Review by zlayabekrija — CHANGES_REQUESTED** [review]
Great improvement, just a few notes: 
Classes `.left` and `.right` are doing the same so keep only one of them. Also as the parent has display flex they don't need flex attribute in this case nor the floats. When sharing the size and using flex elements will space out equally by default. If an element has no content, display flex has no purpose, please remove it. Please get yourself more familiar with display flex and floats.
`.colorBar` class doesn't need margin-top nor does children needs left and top positions, please use floats instead, on children.
Take a look at your code, make these changes, resubmit the new request.
If you need any further help feel free to contact me on slack.
Happy coding.

**Comment by BalintMendli** [comment]
Implemented your suggested changes.
Please let us know if it is ok.

**Review by zlayabekrija — APPROVED** [review]
Great work, great follow up. Pleasure working with you guys.
Thumbs up 👍 
Happy coding.

**Pull request merged**